lib/worker/batcher.ex

```diff
@@ -554,13 +551,22 @@ defmodule BorsNG.Worker.Batcher do
     end
   end
 
-  defp are_reviews_passing(reviews, required) do
-    %{"CHANGES_REQUESTED" => failed, "APPROVED" => passed} = reviews
-
-    case {failed, passed} do
-      {failed, 0} when failed > 0 -> :failed
-      {_, approved} when approved >= required -> :sufficient
-      {0, _} -> :insufficient
+  @spec are_reviews_passing(map, Batcher.BorsToml.t) :: :sufficient | :failed | :insufficient
+  defp are_reviews_passing(reviews, toml) do
+    %{"CHANGES_REQUESTED" => failed, "APPROVED" => approvals} = reviews
+    required? = is_integer(toml.required_approvals)
+    approvals_needed = required? && toml.required_approvals || 0
+    approved? = approvals >= approvals_needed
+    failed? = failed > 0
+    cond do
+      not required? ->
+        :sufficient
```

Comment:
I _really_ want to remove this. I included this clause to keep backwards compatibility (if `required_approvals` is not present, we don't check for failure) but I personally find this strange

Reply:
[Other users asked for it](https://github.com/bors-ng/bors-ng/issues/587). @matklad

I think, at least in the case under discussion, that the user who requested changes was a non-collaborator. If you can make sure that only collaborator's reviews are counted, then it could probably be removed.

Or we could always add a `bors r+ force` option, to more or less mimic GitHub's "turn the merge button red" behavior. But that should probably go through an RFC.

Reply:
Yeah, I would prefer that there be an option to treat bors commands as a single source for whether the PR should be merged. At least in small-scale open-source projects, dismissing requests for changes seems like a busy work. It seems quite unlikely the r+er *accidentally* misses request for changes, and it is quite probably that they want to deliberately ignore it without extra clicks.